src/core/backend/pearl/settings.rs

```diff
@@ -65,7 +57,8 @@ impl Settings {
                 for vdisk_id in vdisks {
                     if let Ok((entry, vdisk_id)) = self.try_parse_vdisk_id(vdisk_id) {
                         if self.mapper.is_vdisk_on_node(&node_name, vdisk_id) {
-                            let disk_name = config.pearl().alien_disk().to_owned();
+                            // name of the disk is not necessary for alien group
+                            let disk_name = String::from("alien_disk");
```

Comment:
I think it's better to leave an option to choose disk for alien folder. If disk is set: alien_full_path = disk_path + alien_path
if not set: alien_full_path = alien_path.